src/core/city-autocomplete.service.js

```diff
@@ -107,4 +107,47 @@ function CityAutocomplete($q, $http, appConfig, UdbPlace, jsonLDLangFilter) {
     return deferredPlaces.promise;
   };
 
+  /**
+   *
+   * Get place by id
+   *
+   * @param {string} id
+   * @returns {Promise}
+   */
+  this.getPlacesById = function(id) {
```

Comment:
When searching a place by id you can get one result at most. Why not just use the `/place/{id}` endpoint in that case?